src/Microsoft.NetCore.Analyzers/Microsoft.NetCore.Analyzers.sarif

```diff
@@ -344,6 +344,25 @@
             ]
           }
         },
+        "CA2013": {
+          "id": "CA2013",
+          "shortDescription": "Do not use ReferenceEquals with value types",
+          "fullDescription": "Value type typed arguments are uniquely boxed for each call to this method, therefore the result is always false.",
```

Comment:
Did you edit this file manually? If so, it needs to be reverted. This is auto-generated file that you can update locally by ruuning `msbuild /v:m /t:pack nuget\NetAnalyzers\Microsoft.CodeAnalysis.NetAnalyzers.Package.csproj` or `eng\common\cibuild.cmd` from the root of the repo. It is fine if these files are not updated as a part of your PR as well, we normally re-generate this periodically.